src/main/java/com/foundation/search/view/ParametersPanel.java

```diff
@@ -29,19 +47,27 @@
  * @version 1.0.
  */
 
-public class ParametersPanel extends JPanel{
+public class ParametersPanel extends JPanel implements ItemListener {
 
-    private JLabel instruction;
-    private JLabel textFilesFound;
-    private JLabel nameLabel;
     private JTextField nameInput;
-    private JLabel pathLabel;
-    private JTextField pathInput;
-    private JLabel extensionLabel;
-    private JTextField extensionInput;
-    private JLabel sizeLabel;
+    private JTextField pathInputLabel;
+    private JFileChooser pathInput;
+    private JButton btnBrowse;
+    private JComboBox extensionInput;
+    private JTextField ownerInput;
+    private JTextField contentInput;
     private JTextField sizeInput;
+    private JComboBox sizeComparison;
+    private JComboBox sizeUnit;
+    private String sizeData[];
     private JButton findButton;
+    private JButton cleanButton;
+    private JCheckBox hidden;
+    private JCheckBox read_only;
```

Comment:
Please rename the name of your JCheckbox and Boolean variables to follow the camelCase format (don't use underscore).
For example, this variable can be: readOnlyCh.

Reply:
done